### ISSUE TITLE
Clicking "Show Details" whilst syncing SD causes Companion to hang

### ISSUE THREAD

**Issue by fraternl** [opentx/opentx#2580]
Clicking "Show Details" whilst syncing SD causes Companion to hang.
Finally (after 5 minutes) it will finish...

![sync](https://cloud.githubusercontent.com/assets/8692856/8914087/78299d4e-349d-11e5-9723-d8e15dc4f592.PNG)

**Comment by fraternl** [comment]
I used "Synchronize SD" today and now on my setup the "Show Details" is checked by default.
I will get the same "not responding", but after a while it comes back and everything is done.

I would make this low priority unless you know of an easy fix.

**Comment by rguptaca** [comment]
I've been using 2.1.1N bills and now 2.1.1 released to the public. On both Windows & and OS X synchronize does not work. On windows it appears to hang. On OS X it shows complete within seconds but nothing actually happens for the source or destination folders.
For Windows I am using an xcopy script for now.
Actually during my testing I removed the SD card form my Taranis and busted it. Maybe I should have disconnected the battery first. Lucky I had made a copy backup and all is good now. 8GB in a Tx is ridiculous:)

**Comment by ReneTr** [comment]
Just to add some information: I used the sync SD today and it also got the Not responding message in Windows 10. However after a couple of seconds it showed what it was doing in really tiny print. (See 1st screenshot)
After a while it completed successfully (I checked and the files did sync) and the final screen was normal. (2nd screenshot)

![screen1 bad](https://cloud.githubusercontent.com/assets/13387411/9185076/899c2cf0-3fb9-11e5-8fb9-70f701ef1f32.png)
![screen 2 good](https://cloud.githubusercontent.com/assets/13387411/9185075/89804c42-3fb9-11e5-8cfd-7ace1999c733.png)
